doc/overview.md

```diff
@@ -126,11 +126,38 @@ scheduled by the Runner step such as building a kernel, running some test,
 performing some static analysis etc.  Each environment needs to have its own
 API token set up locally to be able to submit the results.
 
-### Email Generator
-
-The Email Generator step listens for pub/sub events for when tests are complete
-to then generate a summary and send it via email.  The current implementation
-only prints the summary in the logs, and generates one for every test results
-received.  A production solution would send emails to various recipients
-depending on the tests and would wait for an event signalling that a group of
-tests has completed (or timed out).
+### Email Report
+
+The Email Report in its current status listens for events once a build and its revisions'
+tests are completed and renders an email message using a jinja template, in the rendered
+message you can see relevant information about a build or revision. When the message is ready
+the application opens a connection to an STMP server, and send the report to an user or a list of
+recipients that are waiting for the report with the build information after executing tests and
+possible regressions. With the report it is possible to see the number of tests have failed or passed,
+as seen in the example below:
+
+```mermaid
```

Comment:
As it does not include any diagram, why do we need `mermaid` here?

Reply:
@JenySadadia we do not need, it was wrong ;) all updated.